tests/view/items/test_items.py

```diff
@@ -55,6 +54,15 @@ def test_create(client):
     assert database_item.employee_id == employee.id
 
 
+@pytest.mark.parametrize("path", (
+    "/items/create",
```

Comment:
@vladarefiev mb we will use `url_for` instead of hardcoded urls?

Reply:
@vryazanov can't find solution, i have faced with 
```
RuntimeError: Attempted to generate a URL without the application context being pushed. This has to be executed when application context is available.
```
Do you know smth about this one?

Reply:
@vryazanov fixed